config/config.go

```diff
@@ -14,6 +13,13 @@ type Config struct {
 	TestTargetConfig      TargetConfig  `yaml:"test_target_config"`
 	QueryTweaks           []*QueryTweak `yaml:"query_tweaks"`
 	TestCases             []*TestCase   `yaml:"test_cases"`
+	Timing                Timing        `yaml:"timing"`
+}
+
+type Timing struct {
```

Comment:
I've just been thinking about whether things like this should be a flag or config value, but I guess config value makes sense for now, as all other parameters that affect test execution are also in the config file.

As for naming, how about the bit more elaborate `QueryTimeParameters` or something like that?

By the way, in the future it would be great to run instant queries (not just range queries) as well, and perhaps both types of query as part of the same compatibility test run. But we can figure that out later.

Reply:
Ok I renamed Timing -> QueryTimeParameters.